docs/1.0. Certificate Provisioning.md

```diff
@@ -147,32 +149,33 @@ The EST Server MUST present a valid TLS Server Certificate, this Certificate MUS
 
 ### EST Server - Client Authentication
 
-The EST Server MUST authenticate the EST Client that is requesting a TLS Certificate manually or automatically using a TLS Client Certificate.
+The EST Server MUST authenticate the EST Client that is requesting a TLS (Server?) Certificate manually or automatically using a TLS Client Certificate.
 
 The EST Server MUST support using a TLS Client Certificate, presented during the TLS handshake by the EST Client to authenticate if the EST Client is trusted. The TLS Client Certificate can either be signed by the current CA or a third party-trusted CA. The EST Server MUST check the validity of the EST Client's TLS Certificate before responding to its request, including checking the TLS certificate's revocation status where applicable.
 
-As the TLS Client Certificate is intended to be used for the lifetime of the device, the validity of the TLS Client Certificate and its corresponding chain of trust SHOULD exceed the intended lifetime of the product, it is therefore RECOMMENDED that this is a minimum of 30 years. The EST Server MAY ignore the validity period of the TLS Client Certificate and authenticate an EST Client using an expired TLS Client Certificate, if it has not been revoked.
+As the TLS Client Certificate is intended to be used for the lifetime of the device, the validity of the TLS Client Certificate and its corresponding chain of trust SHOULD exceed the intended lifetime of the product. It is therefore RECOMMENDED that this is a minimum of 30 years. The EST Server MAY ignore the validity period of the TLS Client Certificate and authenticate an EST Client using an expired TLS Client Certificate, if it has not been revoked.
 
-The EST Server MUST provide a method to load multiple trusted Root CA's, that are used to verify the TLS Client Certificate.
+The EST Server MUST provide a method to load multiple trusted Root CAs, that are used to verify the TLS Client Certificate.
 
 The EST Server MAY also support manual authentication of the EST Client if:
 - No TLS Client Certificate is presented during the TLS handshake
 - The TLS Client Certificate is not trusted
 - The local security policy requires an extra authentication step
 
 The exact process for manual authentication will be implementation specific, but the EST Server MUST provide enough information to the operator so they can authenticate the EST Client. During manual authentication the EST Server MUST respond with HTTP 202 and the response MUST include a `Retry-After` header as defined for HTTP 503 responses.
+(Worth linking to RFC 7030 Section 4.2.3 to be clear this is an invention of that spec, not NMOS? I had to check.)
 
 The EST Server MUST not support Basic HTTP Authentication.
```

Comment:
And as opposed to other REQUIREMENTS in this spec, this is new normative language in this spec not just quoted normative language from RFC 7030. Is that right?

Reply:
According to RFC 7030: 

> The EST server can optionally also request that the EST client submit a username/password using the _HTTP Basic_ or Digest authentication method

Should we perhaps remove this line from the spec?

Reply:
@prince-chrismc, @jonathanthorpe-sony, exactly, I agree, unless the activity group has determined a reason to override RFC 7030 or this is already covered by a subsequent RFC or I-D?

Reply:
If this optional feature was enabled it would prevent the workflow we imagined.

Perhaps something like...

```suggestion
The EST Server SHOULD NOT impose Basic HTTP Authentication described in [ref](#)
```

Reply:
I think SHOULD NOT would be suitable.

Reply:
What's the appropriate reference? https://tools.ietf.org/html/rfc7030#section-6 or https://tools.ietf.org/html/rfc7030#section-3.2.3?

Reply:
The paragraphs above REQUIRE the EST Server to support TLS Client Certificates as the means of authentication, at the TLS layer. So do we need to mention HTTP Basic Authentication at all here? Is this one approach to "manual authentication"? @peterbrightwell @prince-chrismc

Reply:
I think it should be left in due to security risk of username/passwords being insecurely stored on devices, but agree it should be replaced with SHOULD NOT.

Reply:
From RFC7030

> For bootstrapping, the EST client can rely upon manual authentication performed by the end-user as detailed in Section 4.1.1.

> the optional certificate-less authentication is used as specified in Section 3.3.3.

🤔 

The TLS authentication MUST be supported and this is optional. 

FRom docs/1.0. Certificate Provisioning.md#L160

> The EST Server MAY also support manual authentication of the EST Client

But this line prevents it.

Let's remove it!

```suggestion
```

Reply:
Manual Auth and Basic HTTP Auth are two different things. 
Basic HTTP Auth is at the Transport layer and would require the EST client to support sending a username & password with the request.
Manual Auth would be something implement on the EST server side, its UI for example to give an admin control to manually approve requests that were not automatically approved, the EST Client would get a HTTP 202 response while waiting for its request to be manually approved

Reply:
OK. RFC 7030 says "The EST server MAY request HTTP-based client authentication" and what you're saying, @JamesGibo, is that we want an additional constraint in BCP-003-03 that "The EST server SHOULD/MUST NOT request HTTP-based client authentication"?

Reply:
Yeah i think we want to add an additional constraint in BCP-003-03 that "The EST Server SHOULD/MUST NOT request HTTP-based client authentication". 
After looking at this again I think, as you said this constraint should cover both Basic and Digest Auth. 
This would prevent EST client implementers from having to support HTTP-based client auth, prevent insecure auth mechanisms and hopefully prevent integration issues

Reply:
Proposal:

The EST Server MUST NOT request HTTP-based client authentication, in order to accommodate EST Clients that do not support the HTTP Basic and Digest Access Authentication mechanisms.

Reply:
Looks good to me

Reply:
I think the commas make that a bit ambiguous. Would prefer `The EST Server MUST NOT request HTTP-based client authentication. This is an optional protocol in [RFC 7030...](...) to accommodate...`

Reply:
Better now!

Reply:
Edited (again) based on @peterbrightwell feedback, that "This" could have referred to "The EST Server MUST NOT request" or to "an optional protocol". Thanks, Pete!